### PR TITLE
Fixed issues with torch 1.5.x and 1.4.0 in test suite / op support

### PR DESCRIPTION
**Pull request by madhavajay**
## Description
Please include a summary of the change, the motivation, and any additionl context that will help others understand your PR. If it closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## Affected Dependencies
None

## How has this been tested?
Locally in Python 3.8 with Torch 1.4.0, 1.5.0, 1.5.1 and 1.6.0

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Comment by madhavajay** [comment]
BTW, first commit has the "black" changes.

File: src/syft/core/io/virtual.py [hunk 1]
```diff
@@ -57,7 +57,9 @@ def _object2proto(self) -> VirtualServerConnection_PB:
     @staticmethod
     def _proto2object(proto: VirtualServerConnection_PB) -> "VirtualServerConnection":
         node = _deserialize(blob=proto.node, from_proto=True)
-        return VirtualServerConnection(node=node,)
+        return VirtualServerConnection(
+            node=node,
```
Comment (Jasopaum) [inline src/syft/core/io/virtual.py:61]: Maybe this could be on one line without the comma?

File: src/syft/core/pointer/pointer.py [hunk 2]
```diff
@@ -137,7 +137,9 @@ def __init__(
         self.tags = tags
         self.description = description
 
-    def get(self,) -> StorableObject:
+    def get(
+        self,
```
Comment (Jasopaum) [inline src/syft/core/pointer/pointer.py:141]: Same, maybe we can remove the comma and make it fit in one line

File: src/syft/proto/lib/torch/parameter_pb2.py [hunk 3]
```diff
@@ -22,7 +22,9 @@
     serialized_options=None,
     create_key=_descriptor._internal_create_key,
     serialized_pb=b'\n\x1fproto/lib/torch/parameter.proto\x12\x0esyft.lib.torch\x1a\x1cproto/lib/torch/tensor.proto"}\n\x0eParameterProto\x12*\n\x06tensor\x18\x01 \x01(\x0b\x32\x1a.syft.lib.torch.TensorData\x12\x15\n\rrequires_grad\x18\x02 \x01(\x08\x12(\n\x04grad\x18\x03 \x01(\x0b\x32\x1a.syft.lib.torch.TensorDatab\x06proto3',
-    dependencies=[proto_dot_lib_dot_torch_dot_tensor__pb2.DESCRIPTOR,],
```
Comment (Jasopaum) [inline src/syft/proto/lib/torch/parameter_pb2.py:25]: Do we really want black to be applied to protobuf files?

File: tests/syft/lib/torch/tensor/tensor_remote_method_api_suite_test.py [hunk 4]
```diff
@@ -30,13 +35,32 @@
...
 
+
+def get_return_type(support_dict: Union[str, Dict[str, str]]) -> str:
+    if isinstance(support_dict, str):
+        return support_dict
+    else:
+        return support_dict["return_type"]
+
+
+def version_supported(support_dict: Union[str, Dict[str, str]]) -> bool:
```
Comment (Jasopaum) [inline tests/syft/lib/torch/tensor/tensor_remote_method_api_suite_test.py:46]: nit: what do you think about naming this one `is_torch_version_supported`?

File: src/syft/lib/torch/__init__.py [hunk 5]
```diff
@@ -14,14 +17,37 @@
...
+        return support_dict
+    else:
+        return support_dict["return_type"]
+
+
+def version_supported(support_dict: Union[str, Dict[str, str]]) -> bool:
+    if isinstance(support_dict, str):
+        return True
+    else:
+        return TORCH_VERSION >= version.parse(support_dict["min_version"])
```
Comment (Jasopaum) [inline src/syft/lib/torch/__init__.py:34]: Maybe these could go into the `allowlist.py` file to avoid duplicated code

**Review by iamtrask — APPROVED** [review]
